static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -198,7 +198,7 @@ function WidgetViewerModal(props: Props) {
     return (
       <React.Fragment>
         {widget.displayType !== DisplayType.TABLE && (
-          <Container>
+          <Container height={widget.displayType !== DisplayType.BIG_NUMBER ? 300 : null}>
```

Comment:
should we have the `300` number in a const?

Reply:
Added a constant for the 300!